client/signup/site-mockup/index.jsx

```diff
@@ -66,18 +66,23 @@ class SiteMockups extends Component {
 		verticalPreviewContent: '',
 	};
 
-	shouldComponentUpdate( nextProps ) {
-		// Debouncing updates to the preview content
-		// prevents the flashing effect.
```

Comment:
What's the "flashing effect" this is preventing? I'm not able to induce something I'd characterize that way.

Reply:
Heh, I could have pulled out the thesaurus on that one :) 

The effect was leaning more towards _'many bouncy fade in and outy'_

https://github.com/Automattic/wp-calypso/pull/32055

Hitting a vertical match mid-sentence, or even two, caused the preview to flutter in and out of the screen like a wayward butterfly!